test/util.py

```diff
@@ -1,11 +1,29 @@
 """
 Some utility functions to help with testing. These mainly add fake objects to use in making sets.
 """
+import json
+
 from installed_clients import FakeObjectsForTestsClient
+from installed_clients.AssemblyUtilClient import AssemblyUtil
 from installed_clients.DataFileUtilClient import DataFileUtil
 
 
-def info_to_ref(info):
+def log_this(config: dict[str, str], file_name: str, output_obj: dict | list) -> None:
+    """Utility function for printing JSON data to a file.
```

Comment:
to replace the numerous print statements that you can't actually see whilst running tests

Reply:
I find it quite helpful when debugging to save a dump of whatever the data is to a file so you can collapse json structures, etc., for easier troubleshooting.